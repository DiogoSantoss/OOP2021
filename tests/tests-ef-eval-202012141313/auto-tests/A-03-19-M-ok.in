5
5
B
300
5
1C6H5OH
1
5
BK
100
0
8
3
P2
B
1
s
4CO2
2
s
C
2
n
3
P1
8B1256
20
n
0
6
1
R1
1
R2
0
0
