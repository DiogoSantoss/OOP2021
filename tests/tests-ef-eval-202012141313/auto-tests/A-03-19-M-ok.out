Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'R1' não existe.
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
BARGAIN|B|300
BARGAIN|1C6H5OH|1
BARGAIN|BK|100
NEW|B|300
NEW|4CO2|16
NEW|C|300
NEW|8B1256|58
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 